### PR TITLE
refactor(tests): remove withData out of testing suites

### PR DESCRIPTION
**Pull request by visylvius**
Removes withData from leche out of all tests,
will submit another PR to remove leche from package.json

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @visylvius has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #664

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/api/uploads/__tests__/MultiputPart-test.js [hunk 1]
```diff
@@ -184,46 +183,24 @@ describe('api/uploads/MultiputPart', () => {
             });
         });
 
-        withData(
-            [
-                [
-                    {
-                        offset: 1,
```
Comment (priyajeet) [inline src/api/uploads/__tests__/MultiputPart-test.js]: This one seems to have 2 objects

**Comment by jstoffan** [comment]
Thanks for contributing!

File: src/api/uploads/__tests__/MultiputPart-test.js (hunk 1 above)
Comment (visylvius) [reply src/api/uploads/__tests__/MultiputPart-test.js]: It was somewhat weird. 

When I logged the contents of the previous test, it came out as one object, and when I tried including the data as two separate objects the test failed.

Comment (visylvius) [reply src/api/uploads/__tests__/MultiputPart-test.js]: Lemme re-try that.

**Pull request merged**